src/Stratis.Bitcoin.Networks/ProtocolVersion/ProtocolVersion.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System.Collections.Generic;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Networks
+{
+    /// <summary>
+    /// Class enumeration containing protocol versions, by name and version number (Id).
+    /// </summary>
+    /// <remarks>
+    /// This class contains baseline versions.
+    /// <para>
+    /// Future versions will go into either Bitcoin <see cref="BitcoinProtocolVersion"/> or Stratis <see cref="StratisProtocolVersion"/>.
+    /// </para>
+    /// </remarks>
+    public class ProtocolVersion : Enumeration
```

Comment:
Probably rename to BaseProtocolVercion or smth and mention in the comments that those are common between stratis and bitcoin

Reply:
Yes, I did have this as `ProtocolVersionBase` originally - I was waiting for feedback.  Cool, I'll rename and add comments accordingly.